SlimeVR/SlimeVR-Server
# Make mounting reset follow headset rotation

It took a bazillion years, but finally I think I can say it's done!!! Writing the tests for this was an enormous pain in the ass, so please if you can, verify the logic behind it. I also tested this quickly and it seemed to work as intended.

Corresponding beta on the Discord: https://discord.com/channels/817184208525983775/1215578136419377183

To test this:

- Ensure manual mounting, full reset, and yaw reset all retain prior functionality.
- Try mounting reset directly after a full reset.
- Try mounting reset directly after a yaw reset.
- Try mounting reset at a different rotation after a full/yaw reset.
